terraform/modules/app/task_definition.tf

```diff
@@ -117,6 +117,9 @@ resource "aws_ecs_task_definition" "bootstrap" {
     },
   )
 
+  lifecycle {
```

Comment:
Might be worth pasting the PR description (basically the whole thing - it's useful context) here as a comment. It's otherwise not obvious that the ignore_changes here is primarily to reduce noisy/meaningless diffs rather than for correctness or some other function reason.

Reply:
Good idea, I've added a comment.